.github/workflows/main-go.yaml

```diff
@@ -24,8 +24,28 @@ permissions:
   contents: read
 
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
+        with:
+          go-version: '1.22'
```

Comment:
Maybe we can do 
```
go-version: '>=1.21.7'
```

Or 
```
go-version: '>=1.22'
```
works - just so we only need to bump this on sec issues

Reply:
Yeah that's a good idea, will change